src/main/java/com/axibase/tsd/client/SelfSignedTrustManager.java

```diff
@@ -0,0 +1,62 @@
+package com.axibase.tsd.client;
+
+import lombok.extern.slf4j.Slf4j;
+
+import javax.net.ssl.TrustManager;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'javax.net.ssl.TrustManager'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.shmagrinsky.com/coding_rules#rule_key=squid%3AUselessImportCheck)